_posts/newsletters/2022-11-04-newsletter.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: post
+title: US-RSE October 2022 Newsletter
+subtitle: October 2022 
+category: newsletter
+tags: [newsletter, supercomputing]
+---
+
+In this monthly newsletter, we share recent, current, and planned activities of the US-RSE Association, and related news that we think is of interest to US-RSE members. Newsletters are also available on our [website](https://us-rse.org/newsletters/) alongside the growing resources and information on the US-RSE Association. To receive our newsletter, join US-RSE [here](https://us-rse.org/join/).
+
+In this issue:
+
+* [Steering Committee Elections](#elections)
+* [First US-RSE Conference in 2023!](#conference2023)
+* [RSE-HPC-2022 Workshop at SC22](#sc22)
+* [US-RSE SC22 Booth](#scbooth)
+* [ADSA/US-RSE Career Workshop](#adsa-usrse)
+* [Help Create a Job Description Archive](#jobarchive)
+* [Steering Committee Recap](#sc-update)
+* [Community News](#news)
+* [Funders Discussion Series](#funders-talk)
+* [RSE Stories Podcast - Now Developer Stories!](#podcast)
+* [November Community Call](#community-call)
+* [Interesting Events](#events)
+* [Interesting Reads, Videos or Podcasts](#reads)
+* [US-RSE Education and Training working group](#education)
+* [US-RSE Outreach](#outreach)
+* [US-RSE Financials](#financials)
+* [Get Involved](#involved)
+* [Recent Job Postings](#jobs)
+
+    
+    
+    
+<a name="elections"></a>
+# **Steering Committee Elections**
+    
+The [yearly election](https://us-rse.org/about/election/) cycle for the Steering Committee is underway.  Information on the candidates will be posted soon.  **Voting will occur December 2-9, 2022.**  US-RSE members were sent an email with the subject "US-RSE Membership Verification" in October confirming their membership.  Everyone who is a [member of US-RSE](https://us-rse.org/join) as of November 20th can vote in the election.  Question about your membership status?  Email membership@us-rse.org.
```

Comment:
```suggestion
The [yearly election](https://us-rse.org/about/election/) cycle for the Steering Committee is underway.  Information on the candidates will be posted soon.  **Voting will occur December 2-9, 2022.**  US-RSE members were sent an email with the subject "US-RSE Membership Verification" in October confirming their membership.  Everyone who is a [member of US-RSE](https://us-rse.org/join) as of November 20th can vote in the election.  Question about your membership status?  Email [membership@us-rse.org](mailto:membership@us-rse.org).
```

Reply:
+1

Reply:
or could make "email" the link text to match below

Reply:
And another link below uses the name of the recipient as link text ....